cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/ImpersonatedTwillPreparer.java

```diff
@@ -47,6 +47,12 @@
   }
 
   @Override
+  public TwillPreparer withConfiguration(Map<String, String> map) {
```

Comment:
It was generated by IntelliJ. Probably because the source was not yet downloaded when this method was generated.